### PR TITLE
[Geeks for Geeks Solution]: Value Equal to Index Value

### PR DESCRIPTION
**Pull request by iamanolive**
resolves issue #337

### CODE REVIEW

File: dsa-solutions/gfg-solutions/_category_.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "label": "LeetCode Solutions",
```
Comment (Ajay-Dhangar) [inline dsa-solutions/gfg-solutions/_category_.json:2]: change lable to `"label": "GFG Solutions",`

File: dsa-solutions/gfg-solutions/0001-value-equal-to-index-value.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+  - Geeks for Geeks
+  - CPP
+  - Python
+  - DSA
+description: "This is a solution to the Value Equal to Index Value problem on Geeks for Geeks."
+---
+
+## Problem Description
+
+Given an array Arr of N positive integers. Your task is to find the elements whose value is equal to that of its index value ( Consider 1-based indexing ).<br />
```
Comment (Ajay-Dhangar) [inline dsa-solutions/gfg-solutions/0001-value-equal-to-index-value.md:18]: remove `<br />`  tag

File: dsa-solutions/gfg-solutions/0001-value-equal-to-index-value.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+
+Final `result` list: `[2]`
+
+## Time Complexity
+
+The time complexity of this solution is O(N) where N is the number of elements in the array. This is because the solution involves a single pass through the array to check each element against its 1-based index.
+
+## Space Complexity
+
+The space complexity is O(1) for the input space, and O(K) for the output space, where K is the number of elements matching the condition. In the worst case, if all elements match, the space used by the result list will be O(N).
```
Comment (Ajay-Dhangar) [inline dsa-solutions/gfg-solutions/0001-value-equal-to-index-value.md:124]: basic practice:

```md
O(1)
```

Good Practice:

```md
$O(1)$
```

and 

basic practice:

```md
O(N)
```

Good Practice:

```md
$O(n)$
```

**Review by Ajay-Dhangar — CHANGES_REQUESTED** [review]
change code based on suggestion

**Comment by iamanolive** [comment]
made the changes @Ajay-Dhangar

**Review by Ajay-Dhangar — APPROVED** [review]
Done

**Pull request merged**